modules/virt-deleting-vm-snapshot-web.adoc

```diff
@@ -13,5 +13,5 @@ You can delete an existing virtual machine (VM) snapshot by using the web consol
 . Navigate to *Virtualization* -> *VirtualMachines* in the web console.
 . Select a VM to open the *VirtualMachine details* page.
 . Click the *Snapshots* tab to view a list of snapshots associated with the VM.
-. Click the options menu {kebab} beside a snapshot and select *Delete VirtualMachineSnapshot*.
+. Click the options menu {kebab} beside a snapshot and select *Delete VirtualMachine snapshot*.
```

Comment:
according to the doc I see sometimes it says: Delete snapshot  and other time Delete VirtualMachine snapshot
@yfrimanm can you tell which it right?

Reply:
@ShellyKa13 It should be `Delete snapshot`.
Thanks for the note.